test/Microsoft.Dnx.Testing.CommonUtils/DnxFunctionalTestBase.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Dnx.Testing
+{
+    public class DnxSdkFunctionalTestBase
+    {
+        public const string SdkVersionForTestingEnvName = "DNX_SDK_VERSION_FOR_TESTING";
+
+        static DnxSdkFunctionalTestBase()
+        {
```

Comment:
I'm not sure why Wei implemented it this way but if I were to guess, I would say that this just ensures that the constructor is only called once. An alternative would be to use a private static flag and checking it inside a non-static constructor. Is that better?